modules/userId/index.js

```diff
@@ -415,6 +444,10 @@ export function init(config) {
       updateSubmodules();
     }
   })
+
+  // exposing getUserIds function in global-name-space so that userIds stored in Prebid can be used by external codes.
+  let theGlobal = getGlobal();
```

Comment:
Avoid creating a reference for the `getGlobal()` value since it's only used once. 
`(getGlobal()).getUserIds = getUserIds`

Reply:
makes sense :)